src/mame/video/realbrk.cpp

```diff
@@ -213,14 +214,17 @@ void realbrk_state::draw_sprites(bitmap_ind16 &bitmap,const rectangle &cliprect,
 		attr    =       s[ 6 ];
 		code    =       s[ 7 ];
 
-		if(( flip & 0x03 ) != layer ) continue;
+		pri     =       flip & 0x03;
+
+		if(pri != layer)
+			continue;
 
 		xnum    =       ((dim >> 0) & 0x1f) + 1;
 		ynum    =       ((dim >> 8) & 0x1f) + 1;
 
 		flipx   =       flip & 0x0100;
 		flipy   =       flip & 0x0200;
-		rot     =       flip & 0x0030;
+		rot     =       (flip & 0x0030) >> 4;
```

Comment:
This one now actively changes implementation as stated before.
Are you sure it's like this?

Reply:
Check the code below - he adjusted the switch statement for the shift.

Reply:
Overlooked it, github sucks lol